.github/workflows/ci.yml

```
@@ -0,0 +1,374 @@
+    timeout-minutes: 30
+    needs: validate
+    
+    strategy:
+      matrix:
+        rust-version: [stable, beta]
+        
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+      
+    - name: Setup Rust toolchain
```

actionlint (1.7.7)

step must run script with "run" section or run action with "uses" section

(syntax-check)